### PR TITLE
Support PyTorch1.7 conversion for CoCosNet and MidasNet models

### PR DESCRIPTION
**Pull request by akorobeinikov**

### CODE REVIEW

File: models/public/midasnet/model.yml [hunk 1]
```diff
@@ -49,6 +49,11 @@ postprocessing:
     file: models/blocks.py
     pattern: 'torch.hub.load\("facebookresearch/WSL-Images", "resnext101_32x8d_wsl"\)'
     replacement: 'models.resnext101_32x8d()'
+  # Replace Squeeze with Gather operation for support PyTorch >= 1.7
```
Comment (IRDonch) [inline models/public/midasnet/model.yml:52]: It's just 1.7, AFAIK. They fixed the Squeeze export in 1.8.

File: models/public/cocosnet/model.yml [hunk 2]
```diff
@@ -97,11 +97,11 @@ postprocessing:
     file: model_files/models/networks/correspondence.py
     pattern: '/ self.opt.down\)'
     replacement: '// self.opt.down)'
-  # MO can only convert Squeeze with fixed dimensions
+  # Replace Squeeze with Gather operation for support PyTorch >= 1.7
```
Comment (IRDonch) [inline models/public/cocosnet/model.yml:100]: I don't think you should just replace the comment like that; the original comment explains _why_ Squeeze is bad in this case.

**Review by IRDonch — COMMENTED** [review]
I guess this works, but are we in a rush to support 1.7? Personally, I would rather wait for 1.8, because then we can actually _remove_ one of these workarounds.

**Comment by eaidova** [comment]
@akorobeinikov I believe we can close this request as pytorch1.8 solves all issues

**Pull request closed without merging**